<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultados_senal</class>
 <widget class="QMainWindow" name="resultados_senal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 247, 245)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tablaFrecSEC">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Canal</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fracuencia (HZ)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Magnitud</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="subtitulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Resultados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_espectro">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Graficar espectro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_desviacion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Desviaci√≥n </string>
    </property>
   </widget>
   <widget class="QLabel" name="Grafica_espectro">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="desviacion">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="atras">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>780</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(159, 183, 190)</string>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_axial">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Grafica del espectro&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_axial_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>670</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Histograma Desviacion&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo_pricipal.qrc"/>
 </resources>
 <connections/>
</ui>
